vuepress/docs/next/tutorials/create/ms/update-data-model.md

```diff
@@ -43,25 +45,26 @@ relationship OneToMany {
    Conference to Session
 }
 ```
-In this case we've added two fields to the Conference entity, introduced the Session entity plus an enum, and added a mapping between the two entities. This is the view you'll get in JDL-Studio for the updated data model.
+The above adds two fields to the Conference entity, introduces the Session entity plus an enum, and creates a mapping between the two entities. Below is the updated data model in JDL-Studio:
 
 ![conference.jdl](./img/jhipster-jdl.png)
 
-4. Now import the jdl file into your application. You may be asked if you want to generate MFEs depended on your options when first generating the project.
+4. Import the JDL file into your application. The options selected during project generation determine if you are prompted with the option to generate MFEs.
```

Comment:
Second sentence, a bit awkward >> The options selected during project generation determine if you are prompted with the 'choice' to generate MFEs. 
Or reverse>> you are prompted with the option to generate MFEs if you selected...